pages/lifejourney/havingAChild.js

```diff
@@ -0,0 +1,10 @@
+import Layout from "../../components/layout";
+
+export default function lifejourney() {
+  return (
+    <Layout
+      bannerTitle="Having a child"
+      bannerText="Lorem ipsum dolor sit amet, consectetur"
+    ></Layout>
+  );
+}
```

Comment:
We _should_ include a new page here, but probably don't need 2 levels of url nesting. Can we just put this page at `pages/having-a-child.js`?

Reply:
No issue changing this, followed the folder structure that Pond was using for the breadcrumbs component but can absolutely change it to the above.